commit f72e079ea252f9677d4c64d2dc201b12639e1001 (refs/changes/72/1372/2)
Author: Julien Gilli <julien.gilli@joyent.com>
Date:   2017-01-30T22:53:00+00:00 (2 years, 8 months ago)
    
    joyent/mdb_v8#84 start working on v1.1.5
    Reviewed by: David Pacheco <dap@joyent.com>
    Approved by: David Pacheco <dap@joyent.com>

diff --git a/version b/version
index 2f1034b..5f84e78 100644
--- a/version
+++ b/version
@@ -8,4 +8,4 @@
 #
 # This value is also used by the "publish" target in the build process.
 #
-1.1.4
+1.1.5
